Bottom: a883fc34b9c2c40f9824baf42576ddcb00500ed7
Top:    d86a88924781996b69e21c47534b6bdac7c313f0
Author: scott snyder <sss@karma>
Date:   2020-06-01 00:12:07 -0400

Cyanide & Happiness: Fix for spaces in url (such as in 5572).



---

diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/Cyanide.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/Cyanide.java
index c93fd75..230adb2 100755
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/Cyanide.java
+++ b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/Cyanide.java
@@ -148,6 +148,7 @@ public class Cyanide extends RandomIndexedComic {
 		Log.d(Cyanide, "In parse, final_prev = "+final_prev);		
 		final_str = final_str.replaceAll(".*og:image\" content=\"","");
 		final_str = final_str.replaceAll("\".*","");
+		final_str = final_str.replaceAll(" ","%20");
 		final_title = final_title.replaceAll(".*net/comics/","");
 		final_title = final_title.replaceAll("/\".*","");
 		strip.setTitle(final_title);
